openapi/listings/listings.yaml

```diff
@@ -447,3 +601,297 @@ components:
+      x-examples:
+        Example 1:
+          - hoursTypeId: general
+            regularHours:
+              - openDay: wednesday
+                closeDay: wednesday
+                openTime: '09:45'
+                closeTime: '09:45'
+            specialHours:
+              - status: SPECIAL_HOURS_STATUS_CLOSED
+                startDate: '2019-08-24'
+                endDate: '2019-08-24'
+                openTime: '09:45'
+                closeTime: '09:45'
+      description: |-
+        The operating hours for the listing profile. 
+
+        Each entry in the array is a set of hours for a particular department or customer of the business. It is recommended to provide the `general` hours for all locations. Some locations may have additional definitions for 
+      items:
+        type: object
+        properties:
+          hoursTypeId:
+            type: string
+            description: |-
+              The category of hours that may apply to a specific department or service. Not all types are valid for all business categories.
+
+              The general hours for a location must be set before defining hours for other departments. 
+
+              Currently only `general` is supported in the Vendasta platform. We intend to add others in the future.
+            enum:
+              - general
+              - delivery
+              - kitchen
+              - pickup
+              - driveThrough
+              - takeout
+          regularHours:
+            type: array
+            description: |-
+              A collection of times that this location is open for business. Each period represents a range of hours when the location is open during the week.
+
+              You may specify multiple entries for the same date if you are open for multiple periods (Example lunch and supper).
+            items:
+              type: object
+              properties:
+                openDay:
+                  allOf:
+                    - type: string
+                      x-stoplight:
+                        id: 3u07tkjf0bxdg
+                      description: Indicates the day of the week that this period starts on.
+                    - $ref: '#/components/schemas/dayOfTheWeek'
+                closeDay:
+                  allOf:
+                    - type: string
+                      x-stoplight:
+                        id: svnvxunjr767f
+                      description: Indicates the day of the week that this period ends on. It should be the same as `openDay` except when the period goes over midnight. The total period may not be more than 24 hours
+                    - $ref: '#/components/schemas/dayOfTheWeek'
+                openTime:
+                  $ref: '#/components/schemas/timeOfDay'
+                closeTime:
+                  $ref: '#/components/schemas/timeOfDay'
+              required:
+                - openDay
+                - openTime
+                - closeTime
+          specialHours:
```

Comment:
We should add a description to the specialHours explaining what impact adding records to it has on the regular hours. I'm guessing that adding an `open` specialHours overrides the entire day. What does adding a `closed` record do? What if you have both `open` and `closed` hours on the same day?

Reply:
Based on the revised descriptions, I'm still not clear on the answer to last two questions.